esmf/meta.yaml

```diff
@@ -7,14 +7,18 @@ source:
     git_tag: ESMF_7_0_0
 
 build:
-    number: 0
+    number: 1
     skip: True  # [win]
 
 requirements:
     build:
+        - gcc
         - netcdf-fortran
+        - libnetcdf 4.2.1.1
     run:
+        - libgcc
         - netcdf-fortran
+        - libnetcdf 4.2.1.1
```

Comment:
For some weird reason conda still tries to update `libnetcdf` even though `netcdf-fortran` has both `build` and `run` dependencies set to `libnetcdf 4.2.1.1`.